specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -1913,6 +1913,84 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/containerlogs": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Gets the last lines of docker logs for the given site",
+        "description": "Gets the last lines of docker logs for the given site",
+        "operationId": "WebApps_GetWebSiteContainerLogs",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of web app.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "file"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/containerlogs/zip": {
```

Comment:
missing segments. containerLogs is the resource type as per the signature. So you need a resourceTypename segment on which you will call zip action. A better way to model this API would be return a pointer to storage blob where the zip can be downloaded from. We generally avoid content other than application/json flowing through ARM.